### DISCUSSION TITLE
Where can I access the v10 docs?

### DISCUSSION THREAD

**Discussion by EduardoLopes** [carbon-design-system/carbon#11344] [Help] [answered]
There's any place I can access the v10 docs?

[v11 still doesn't support typescript](https://github.com/carbon-design-system/carbon/discussions/10752), and this is the only thing that is holding my team from migrating to the new version. So we still need to check the v10 docs.

**Comment by tay1orjones** [comment, marked as answer]
Hey @EduardoLopes, yes v10 docs can be found at:

- https://v10.carbondesignsystem.com
- https://v7-react.carbondesignsystem.com/

**Reply by EduardoLopes** [reply]
Thank you so much! ✨

**Reply by tay1orjones** [reply]
Welcome! I just noticed that the storybook needs to be redeployed for the newest version `7.57.1`, I'll work on getting that fixed.
